CHANGELOG.md

```diff
@@ -13,6 +13,11 @@ Mapbox welcomes participation and contributions from everyone.
 - Marked `InfoPanelBinder`, `NavigationView`, `NavigationViewApi` methods with `@UiThread` annotation. [#6269](https://github.com/mapbox/mapbox-navigation-android/pull/6269)
 - Marked `MapboxExitText`, `MapboxLaneGuidance`, `MapboxLaneGuidanceAdapter`, `MapboxManeuverView`, `MapboxPrimaryManeuver`, `MapboxSecondaryManeuver`, `MapboxStepDistance`, `MapboxSubManeuver`, `MapboxTurnIconManeuver`, `MapboxUpcomingManeuverAdapter` methods with `@UiThread` annotation. [#6270](https://github.com/mapbox/mapbox-navigation-android/pull/6270)
 - Marked `RouteShieldCallback` methods with `@UiThread` annotation. [#6271](https://github.com/mapbox/mapbox-navigation-android/pull/6271)
+- Updated alternatives route processing. `MapboxNavigation#setNavigationRoutes` doesn't accept alternatives which don't deviate from primary route or which have different destinations. Not accepted alternatives will be present in `RoutesSetSuccess.ignoredAlternatives`. [#6464](https://github.com/mapbox/mapbox-navigation-android/pull/6464)
+- Fixed alternatives removing. [#6464](https://github.com/mapbox/mapbox-navigation-android/pull/6464)
```

Comment:
Could you elaborate here? This might not be clear enough to developers.

Reply:
it's an artefact after rebase, removed